app/app/src/main/kotlin/com/hedvig/app/feature/profile/ui/aboutapp/AboutAppDestination.kt

```diff
@@ -0,0 +1,170 @@
+package com.hedvig.app.feature.profile.ui.aboutapp
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.WindowInsetsSides
+import androidx.compose.foundation.layout.asPaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.only
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.safeDrawing
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Icon
+import androidx.compose.material3.LocalContentColor
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.google.accompanist.web.WebView
+import com.google.accompanist.web.rememberWebViewState
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.icons.Hedvig
+import com.hedvig.android.core.icons.hedvig.normal.ChevronRight
+import com.hedvig.android.core.ui.appbar.TopAppBarWithBack
+import com.hedvig.app.BuildConfig
+import com.hedvig.app.feature.embark.ui.MemberIdViewModel
+import hedvig.resources.R
+
+private const val licensesUrl = "file:///android_asset/open_source_licenses.html"
+
+@Composable
+internal fun AboutAppDestination(
+  viewModel: MemberIdViewModel,
+  onBackPressed: () -> Unit,
+  isProduction: Boolean,
+) {
+  val memberIdState by viewModel.state.collectAsStateWithLifecycle()
+  val memberId: String? by remember {
+    derivedStateOf {
+      when (val latestMemberIdState = memberIdState) {
+        is MemberIdViewModel.State.Success -> latestMemberIdState.id
+        else -> null
+      }
+    }
+  }
```

Comment:
I think since here the result (memberId) is a simple mapping of what the status of `memberIdState` is, we can skip the remember and just do

```suggestion
  val memberId: String? = when (val latestMemberIdState = memberIdState) {
    is MemberIdViewModel.State.Success -> latestMemberIdState.id
    else -> null
  }
```
derivedStateOf usually is helpful when the state onbject changes often, but the result of whatever is inside the calculation does not change as often, which I don't think is the case here.